python/Ganga/GPIDev/Lib/File/MassStorageFile.py

```diff
@@ -101,7 +101,8 @@ def _setNamePath(self, _namePattern='', _localDir=''):
     def _on_attribute__set__(self, obj_type, attrib_name):
         # This is defining the object as uncopyable from outputfiles... do we want this mechanism still?
         r = copy.deepcopy(self)
-        if _getName(obj_type) == 'Job' and attrib_name == 'outputfiles':
+        from Ganga.GPIDev.Lib.Job.Job import Job
```

Comment:
Can this import be put at the top of the file? It should be possible since `Job` shouldn't be importing this file.